tck/optional/src/main/java/org/eclipse/microprofile/metrics/test/optional/MetricAppBeanOptional.java

```diff
@@ -222,4 +224,60 @@ public String  getMultipleParam4(@QueryParam("qp1") Set<String> v1,
     public String  getNameObject(@QueryParam("qp1") NameObject v1) throws Exception {
         return "This is a GET request with NameObject";
     }
+    
+    @GET
+    @Path("/get-async")
+    public void  getAsync(@Suspended final AsyncResponse ayncResponse) throws Exception {
+        Thread thread = new Thread ( () -> {
+            try {
+                Thread.sleep(5000);
+            } 
+            catch (Exception e) {
+                System.err.println(e.toString());
+            }
+        });
+        thread.run();
+        
+        ayncResponse.resume("This is a GET request with AsyncResponse");
+    }
+    
+    
+    @GET
+    @Path("/post-multiple-param1")
+    public String  postMultipleParam1(
+            @QueryParam("qp1") boolean v1,
+            @QueryParam("qp2") int v2,
+            @QueryParam("qp3") double v3,
+            @QueryParam("qp4") String v4,
+            @QueryParam("qp5") long v5) throws Exception {
+        return "This is a POST request with multiple parameters1";
```

Comment:
annotation says it's a GET request (same issue below too).

Reply:
fixed :(